packages/react-resizable-panels/src/PanelResizeHandle.test.tsx

```diff
@@ -27,7 +27,7 @@ describe("PanelResizeHandle", () => {
   let container: HTMLElement;
 
   beforeEach(() => {
-    // @ts-expect-error
+    // @ts-expect-error globalThis is not typed
```

Comment:
This comment seems kind of confusing. The code isn't reference `globalThis`.

I'm also kind of curious why every `global.IS_REACT_ACT_ENVIRONMENT` comment is slightly different in this PR 😆

Reply:
> I'm also kind of curious why every `global.IS_REACT_ACT_ENVIRONMENT` comment is slightly different in this PR 😆

Since this is opinionated I actually wanted your review on what I should write for these (I also may or may not have let AI write those for me 😅)